build/helper/metadata_add_all.py

```diff
@@ -106,7 +106,10 @@ def _add_ctypes_variable_name(parameter):
 
 def _add_ctypes_type(parameter, config):
     '''Adds a ctypes_type key/value pair to the parameter metadata for calling into the library'''
-    parameter['ctypes_type'] = parameter['type']
+    if parameter['type'] == 'ViAddr' and (parameter['use_array'] or parameter['use_list']):
```

Comment:
await: I don't agree that this hack is legitimate. The change must happen in the hapigen metadata to override the gRPC information. 

Why? Look at this 'export' structure from the .proto file:
```
message ExportAttributeConfigurationBufferResponse {
  int32 status = 1;
  repeated fixed64 configuration = 2;
}
```
It's not `repeated fixed64`, it's `bytes`.

Reply:
As you can see, 'bytes' is [what is checked into gRPC device](https://github.com/ni/grpc-device/blob/main/generated/nidmm/nidmm.proto#L952).

Reply:
Fixed the proto.

But the hack is still necessary, for the LibraryInterpreter code. I agree that the change here is not ideal, but until [AB#2185120](https://dev.azure.com/ni/DevCentral/_workitems/edit/2185120) is fixed, this is the best I could come up with. I would've preferred to do it in the hapigen data, but the code here ignores the python_type, and the PythonExpander in hapigen only looks at the cheader type. I could've modified the one of those things, of course, but those would've been larger changes that both seem like overkill for this little use case, especially since we've got that Tech Debt to change how these work.